packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -70,6 +71,10 @@ function get_base_diffusers_model(model: ModelData): string {
 	return model.cardData?.base_model?.toString() ?? "fill-in-base-model";
 }
 
+function get_prompt_from_diffusers_model(model: ModelData): string | undefined {
+	return (model.widgetData?.[0] as WidgetExampleTextInput).text ?? model.cardData?.instance_prompt ?? undefined;
```

Comment:
```suggestion
	return (model.widgetData?.[0] as WidgetExampleTextInput).text ?? model.cardData?.instance_prompt;
```

no need for explicit `undefined`